### DISCUSSION TITLE
Use standard resource formats

### DISCUSSION THREAD

**Discussion by VectorWorkshopBaoErJie** [godotengine/godot-proposals#5864] [Engine Core]
It is recommended that images, models, music and other materials be processed into standard resource formats through independent resource processing editing tools before being imported into godot, rather than imported into the project. In this way, there is no need to scan for changes in file resources such as png images; There is room for further optimization of resource formats, such as resource packs; Standard resources can be reused across multiple projects.

**Comment by Calinou** [comment]
Godot uses a resource importing system, as standard resource formats are notoriously bad at storing metadata information, which specifies *how* textures should be used (e.g. compression mode). Also, VRAM compression requires an expensive compression step, which shouldn't be done at load-time in a modern game due to how much time it'd take.

> Standard resources can be reused across multiple projects.

This is already feasible, even with the imported files themselves :slightly_smiling_face: 

If you really want to skip the importing step, you can use `.res` files directly as textures or whatever these files have been imported to.
